specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json

```diff
@@ -242,7 +242,8 @@
           "Successful query": {
             "$ref": "./examples/questionanswering/authoring/SuccessfulProjectDeleteJobStatus.json"
           }
-        }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
Shouldn't this be added to the "delete" operation, not the "get" operation?

Reply:
Sorry, by mistake. Fixed it. Thanks for that.